automod/triggers.go

```diff
@@ -1034,9 +1073,16 @@ func (spam *SpamTrigger) CheckMessage(triggerCtx *TriggerContext, cs *dstate.Cha
 
 		if strings.ToLower(strings.TrimSpace(v.Content)) == mToCheckAgainst {
 			count++
-		} else {
-			break
+			continue
 		}
+
+		// this if statement makes me want to cry
+		if settingsCast.SanitizeText && strings.ToLower(strings.TrimSpace(common.SanitizeText(v.Content))) == mToCheckAgainst {
```

Comment:
If this statement makes you cry then make it simpler? why do you want to write oneliners? Create variables  with computed information and then use them here.

Reply:
True

Reply:
Also, please only add comments that are descriptive of your code and not your emotions. This isn't your personal project.

Reply:
Yeah I mostly put that there so someone would comment specifically on it